packages/dataviews/build-style.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+const { promisify } = require( 'util' );
+const fs = require( 'fs' );
+const path = require( 'path' );
+const sass = require( 'sass' );
+const postcss = require( 'postcss' );
+
+/**
+ * Path to packages directory.
+ *
+ * @type {string}
+ */
+const PACKAGES_DIR = path
+	.resolve( __dirname, '../../packages' )
+	.replace( /\\/g, '/' );
+
+/**
+ * Promisified fs.readFile.
+ *
+ * @type {Function}
+ */
+const readFile = promisify( fs.readFile );
+
+/**
+ * Promisified fs.writeFile.
+ *
+ * @type {Function}
+ */
+const writeFile = promisify( fs.writeFile );
+
+/**
+ * Promisified fs.writeFile.
+ *
+ * @type {Function}
+ */
+const makeDir = promisify( fs.mkdir );
```

Comment:
This is interesting, the code apparently predates the `fs/promises` module 🙂

Reply:
Yeah, this has been extracted from [build-workers](https://github.com/WordPress/gutenberg/blob/trunk/bin/packages/build-worker.js) as it is. Most of it has been written a few years ago, and I suppose there could be simpler ways to do the same today. I didn't think it was worth refactoring this, as it's convenient to be able to compare with upstream.